packages/widgets/src/health-monitoring/component.tsx

```diff
@@ -42,267 +47,342 @@ export default function HealthMonitoringWidget({
-                            ? `${(disk.temperature * 1.8 + 32).toFixed(1)}°F`
-                            : `${disk.temperature}°C`}
-                        </Text>
-                      </Group>
-                      <Group gap="1cqmin">
-                        <IconFileReport className="health-monitoring-disk-status-icon" size="5cqmin" />
-                        <Text className="health-monitoring-disk-status-value" size="4cqmin">
-                          {disk.overallStatus}
-                        </Text>
-                      </Group>
-                    </Flex>
-                    <Progress.Root className="health-monitoring-disk-use" size={progressSize}>
-                      <Tooltip label={disk.used}>
-                        <Progress.Section
-                          value={disk.percentage}
-                          color={progressColor(disk.percentage)}
-                          className="health-monitoring-disk-use-percentage"
-                        >
-                          <Progress.Label className="health-monitoring-disk-use-value">
-                            {t("widget.healthMonitoring.popover.used")}
-                          </Progress.Label>
-                        </Progress.Section>
-                      </Tooltip>
+                        </Group>
+                      </Flex>
+                      <Progress.Root className="health-monitoring-disk-use" size={progressSize}>
+                        <Tooltip label={disk.used}>
+                          <Progress.Section
+                            value={disk.percentage}
+                            color={progressColor(disk.percentage)}
+                            className="health-monitoring-disk-use-percentage"
+                          >
+                            <Progress.Label className="health-monitoring-disk-use-value">
+                              {t("widget.healthMonitoring.popover.used")}
+                            </Progress.Label>
+                          </Progress.Section>
+                        </Tooltip>
 
-                      <Tooltip
-                        label={
-                          Number(disk.available) / 1024 ** 4 >= 1
-                            ? `${(Number(disk.available) / 1024 ** 4).toFixed(2)} TiB`
-                            : `${(Number(disk.available) / 1024 ** 3).toFixed(2)} GiB`
-                        }
-                      >
-                        <Progress.Section
-                          className="health-monitoring-disk-available-percentage"
-                          value={100 - disk.percentage}
-                          color="default"
+                        <Tooltip
+                          label={
+                            Number(disk.available) / 1024 ** 4 >= 1
+                              ? `${(Number(disk.available) / 1024 ** 4).toFixed(2)} TiB`
+                              : `${(Number(disk.available) / 1024 ** 3).toFixed(2)} GiB`
```

Comment:
I think it would be best to use "humanFileSize" here and in other instances that have a "GiB" or "MiB" or anything of the likes.